### PR TITLE
chore(github): Update issue submission templates

### PR DESCRIPTION
**Pull request by bryplano**
## What Changed

- Added a `feature-request` Github template
- Updated the `bug-report` template for cleanliness
- Both templates now add a basic title to help identify type of issue quickly

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+assignees: ''
+
+---
+
+# Feature Request
+
+## Describe the Feature Request
+<!-- A clear and concise description of what the feature request is. Please include if your feature request is related to a problem. -->
+
+## Platform Support Requested
```
Comment (jcesarmobile) [inline .github/ISSUE_TEMPLATE/feature-request.md]: there are no boxes to check

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -1,39 +1,48 @@
...
+title: 'bug: '
 labels: ''
 assignees: ''
 
 ---
 
-Description of the problem:
+# Bug Report
+
+## Capacitor Version
```
Comment (jcesarmobile) [inline .github/ISSUE_TEMPLATE/bug-report.md]: maybe tell them to run `capacitor doctor` and paste the whole `Installed Dependencies:` section, as capacitor is made by a lot of packages and they might have the CLI updated but not the platforms

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -1,39 +1,48 @@
...
+<!-- Provide the version of Capacitor you are using in your application. This can be found in your package.json file. -->
+
+## Affected Platform(s)
```
Comment (jcesarmobile) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Maybe add the checkbox comment you have in the feature request template, and tell them to leave empty and check after creating the issue, I've seen a lot of people struggle with this and making invalid checkboxes or deleting the options they don't want

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -1,39 +1,48 @@
...
 
-Other information:
+## Expected Behavior
+<!-- Describe what the behavior would be without the bug. -->
 
-Capacitor version:
+## Sample Code or Sample Application Repo
 
-node version:
+<!-- If you are able to illustrate the bug or feature request with an example, please provide sample code snippets or a sample application via a public GitHub or Bitbucket repo. -->
```
Comment (jcesarmobile) [inline .github/ISSUE_TEMPLATE/bug-report.md]: maybe add some note about issues with sample projects take priority as they are easier to reproduce

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (bryplano) [reply .github/ISSUE_TEMPLATE/bug-report.md]: ??? `capacitor doctor`? Is that a thing?

Comment (jcesarmobile) [reply .github/ISSUE_TEMPLATE/bug-report.md]: yeah, but should be executed with npx ( `npx cap doctor` or `npx capacitor doctor`), I forgot I have a global capacitor for testing, but for regular users it won't work without npx

**Comment by bryplano** [comment]
@jcesarmobile - Updated, let me know if the changes suffice :)

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+<!-- Check one or more boxes for the platform(s) you would like this feature to support. Note that some features may not be feasible for a particular platform (i.e. no web API may be available). -->
```
Comment (jcesarmobile) [inline .github/ISSUE_TEMPLATE/feature-request.md]: Should be similar to the bug note (`<!-- Leave these checkboxes EMPTY until after the issue is created. Once the issue is created, check one or more boxes for the platform(s) this bug affects. -->`)

Comment (bryplano) [reply .github/ISSUE_TEMPLATE/feature-request.md]: Fixed :)

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+---
+
+# Feature Request
+
+## Describe the Feature Request
+<!-- A clear and concise description of what the feature request is. Please include if your feature request is related to a problem. -->
+
+## Platform Support Requested
+<!-- Leave these checkboxes EMPTY until after the issue is created. Once the issue is created, check one or more boxes for the platform(s) this bug affects. -->
```
Comment (jcesarmobile) [inline .github/ISSUE_TEMPLATE/feature-request.md]: I copied the bug text for reference, but this one should say feature request instead

Comment (bryplano) [reply .github/ISSUE_TEMPLATE/feature-request.md]: Oops, copypasta error. My bad.

**Pull request merged**